src/main/java/com/deusgmbh/xcusatio/context/wildcard/CalendarContext.java

```diff
@@ -0,0 +1,56 @@
+package com.deusgmbh.xcusatio.context.wildcard;
+
+import com.deusgmbh.xcusatio.data.lecturer.Lecturer;
+
+public class CalendarContext {
+    private LectureEvent lectureEvent;
+    private String minutesLeft;

```

Comment:
statt hier den string abzuspeichern, könntest du bitte stattedessen ein format wählen, dass es erlaubt berechnungen mit der Zeit zu machen. Also entweder die sekunden angeben oder in irgendeiner Form die Date klasse von Java nutzen.

Und dann halt einmal eine getMinutesLeft methode, welche den echten wert zurückgibt und einmal eine getMinuesLeftText oder soetwas Alternative könnte man auch eine funktion bereitstellen, die aus einem beliebigen Wert dieser Art dann einen String im Zeit Format machen kann.